### ISSUE TITLE
Uniswap - Getting unhandled error when using USDC.e

### ISSUE THREAD

**Issue by rapcmia** [hummingbot/gateway#143]
**Describe the bug**
While checking uniswap connector specifically `arbitrum_one`, we observed that when you use `USDC.e` as your token and managed to get a trade opportunity using amm-arbirage the client would return an error of `Unhandled error in background task: 'USDC.e'`

![image](https://github.com/hummingbot/gateway/assets/73840223/7ec8761f-2c88-4f75-ac17-068c64654564)

```
2023-07-03 20:21:38,001 - 13734 - hummingbot.core.utils.async_utils - ERROR - Unhandled error in background task: 'USDC.e'
Traceback (most recent call last):
  File "/home/rapcmia/github/hummingbot/dev/hummingbot/core/utils/async_utils.py", line 9, in safe_wrapper
    return await c
  File "/home/rapcmia/github/hummingbot/dev/hummingbot/strategy/amm_arb/amm_arb.py", line 232, in main
    await self.apply_slippage_buffers(profitable_arb_proposals)
  File "/home/rapcmia/github/hummingbot/dev/hummingbot/strategy/amm_arb/amm_arb.py", line 258, in apply_slippage_buffers
    arb_side.amount = market.quantize_order_amount(arb_side.market_info.trading_pair, arb_side.amount)
  File "hummingbot/connector/connector_base.pyx", line 481, in hummingbot.connector.connector_base.ConnectorBase.quantize_order_amount
    return self.c_quantize_order_amount(trading_pair, amount)
  File "hummingbot/connector/connector_base.pyx", line 474, in hummingbot.connector.connector_base.ConnectorBase.c_quantize_order_amount
    order_size_quantum = self.c_get_order_size_quantum(trading_pair, amount)
  File "hummingbot/connector/connector_base.pyx", line 453, in hummingbot.connector.connector_base.ConnectorBase.c_get_order_size_quantum
    return self.get_order_size_quantum(trading_pair, order_size)
  File "/home/rapcmia/github/hummingbot/dev/hummingbot/connector/gateway/amm/gateway_evm_amm.py", line 838, in get_order_size_quantum
    return max(self._amount_quantum_dict[base], self._amount_quantum_dict[quote])
KeyError: 'USDC.e'
```

While this is true that is returning an error, a workaround is to make sure to use `USDC.E` instead, below is a screenshot that works and able to arbitrage
![image](https://github.com/hummingbot/gateway/assets/73840223/6d63cb24-baf9-4a56-9ff9-75201d7c3aa5)

**Steps To Reproduce**
1. Setup latest development for both gateway and client
2. Connect uniswap arbitrum wallet
3. Setup simple AMM arb and observe behavior when setting market either `USDC.e` or `USDC.E`

**Release version**
Development 1.18.0, client + gateway 

**Attachments**
[logs_amm-arb.log](https://github.com/hummingbot/gateway/files/11936940/logs_amm-arb.log)
[logs_gateway_app.log - Copy.2023-07-03.log](https://github.com/hummingbot/gateway/files/11936945/logs_gateway_app.log.-.Copy.2023-07-03.log)
[amm-arb.yml.log](https://github.com/hummingbot/gateway/files/11936950/amm-arb.yml.log)

**Comment by fengtality** [comment]
## Bounty

* Sponsor: Hummingbot Foundation
* Budget: [HGP-26](https://snapshot.org/#/hbot.eth/proposal/0x7a02f532e0452d3ce7f8b64a8a52e6bed285cc65461a1523fa0f91f2024f8c01)
* Classification: P3
* Bounty: 10,000 HBOT
* Developer share: 100% - paid upon merge to development
* Status: OPEN

**Comment by MAWUT0R** [comment]
i'm new to working bounties here. can i pick this up?

**Comment by isreallee82** [comment]
Got a `fix `on this while working on #144 



![Image](https://github.com/hummingbot/gateway/assets/70828114/7292403d-306a-4eef-b034-0e08cbe43f32)

Due to the api call to a wrong token symbol USDC.E instead of USDC.e

- refer to #144 for more information

**Comment by nikspz** [comment]
for the mention https://github.com/hummingbot/gateway/pull/177

**Comment by isreallee82** [comment]
![Image](https://github.com/hummingbot/gateway/assets/70828114/0f2afb21-c2dc-4736-84d1-3e8d4dd0b8b9)

**Comment by nikspz** [comment]
Closing this bounty as paid, fix provided in 1.19.0 release
